src/hyperv/hyperv_api_v2.c

```diff
@@ -1716,22 +1715,19 @@ hyperv2DomainAttachSyntheticEthernetAdapter(virDomainPtr domain,
     VIR_FREE(params);
     virBufferFreeAndReset(&query);
 
-    if (hyperv2MsvmComputerSystemFromDomain(domain, &computer) < 0)
-        goto cleanup;
-
     /* build the two __PATH variables */
     if (virAsprintf(&switch__PATH, "\\\\%s\\root\\virtualization\\v2:"
                 "Msvm_VirtualEthernetSwitch.CreationClassName="
                 "\"Msvm_VirtualEthernetSwitch\",Name=\"%s\"",
-                computer->data->ElementName, net->data.network.name) < 0)
+                hostname, vSwitch->data->Name) < 0)
```

Comment:
yes,  it's pulled from Msvm_Computersystem->data->ElementName by the caller and passed to this one as hostname